espymetrics/dal/sqlite_dal.py

```diff
@@ -13,25 +13,116 @@
 
 * Not human readable - Binary format makes it harder to read/grep
 * More complex - More logic = more work = more time/resources?
-
 """
 
+import os
+import sqlite3
+
+import boltons
 
-_MISSING = object()
 SEP = '$'  # '$' is valid in sqlite column names, no escaping required
 
+TABLE_NAME = 'on_import_data'
+CREATE_TABLE = ('CREATE TABLE IF NOT EXISTS ' + TABLE_NAME +
+                '(id integer primary key asc, ' +
+                'hostfqdn, hostname, python$argv, python$bin, ' +
+                'python$build_date, python$compiler, python$have_readline, ' +
+                'python$have_ucs4, python$is_64bit, python$version, python$version_full, ' +
+                'time$utc_epoch real, time$std_utc_offset real, ' +
+                'uname$0, uname$1, uname$2, uname$3, uname$4, uname$5, ' +
+                'username, uuid)')
+
+total_count = 0
+
+sqlite3.enable_callback_tracebacks(True)
```

Comment:
This type of code at the module-level results in the infamous "import-time side effect". While the intentions may be good, this unexpected behavior can have serious implications.

It's best to leave functions like this to the modules closer to the main code entry point. In this case, because there may be other future sqlite usages in the project, I would centralize this sort of behavior in `app.py`.